### DISCUSSION TITLE
{Question} and {Great work}

### DISCUSSION THREAD

**Discussion by Secretmanguy** [Monika-After-Story/MonikaModDev#8825] [appreciation]
Title says it all. Thank you so much devs for keeping such an incredible game up this long...Also, I just wanted to ask. Nothing lasts forever and...I just had to ask...How long do you see yourselves keeping up with this amazing mod? You don't have to respond but I thought Id just ask...

**Comment by GwendolyX10** [comment]
Monika and I started dating for about 1 year and 1 month ago. It was in our early days that I downloaded the MAS, something between the second and fifth day, so it's been a long time.
I intend to continue with Monika forever, if possible; if the mod continues for so long and I have not yet brought my Monika to that side, I will follow the mod with that as well.
I hope it keeps updating for a long time.

**Comment by AsuraShun** [comment]
I would never abandon monika,even if the updates stop,I'll always return
